src/main/java/de/doubleslash/keeptime/App.java

```diff
@@ -218,10 +219,10 @@ private void initialisePopupUI(final Stage primaryStage) throws IOException {
       LOG.debug("Initialising popup UI.");
 
       globalScreenListener = new GlobalScreenListener();
```

Comment:
My idea was to put the if(!OS.isLinux()) switch to the creation of the Listener. If we are on linux, we do not need to create an instance of it :)
- So the change in the GlobalScreenListener class would not be needed.
- The changes in this class would need to validate globalScreenListner!=null insted of the linux switch.

Reply:
reverted the changes in GlobalScreenListener